crates/services/p2p/src/peer_manager.rs

```diff
@@ -497,13 +500,13 @@ impl NetworkBehaviour for PeerManagerBehaviour {
 pub struct PeerInfo {
     pub peer_addresses: HashSet<Multiaddr>,
     pub client_version: Option<String>,
-    pub latest_ping: Option<Duration>,
```

Comment:
would it be useful to track both block height and duration since last heartbeat?

Reply:
yeah, especially if we gonna protect it against ddos

Reply:
done